conformance/system_macros/parse_ion.ion

```diff
@@ -0,0 +1,26 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+(ion_1_1 "parse_ion can be invoked"
+          (each "in text with an unqualified macro name"
+                (text ''' (:parse_ion "true") ''')
+                "in text with an unqualified macro address"
+                (text ''' (:16 "true") ''')
+                "in text with a qualified macro name"
+                (text ''' (:$ion::parse_ion "true") ''')
+                "in text with a qualified macro address"
+                (text ''' (:$ion::16 "true") ''')
+                "in binary with a system macro address"
+                (binary "EF 10 94 74 72 75 65")
+                "in binary with a user macro address"
+                (binary "10 94 74 72 75 65")
+                (produces true)))
+
+// parse_ion can read Ion 1.0 and Ion 1.1, text and binary
+
+// parse_ion always produces user values
+//  - but if the result passes through other macros it could turn it into a system value...?
```

Comment:
Yes. And that's ok because the other macros are allowed to affect the current encoding context while the values in the nested stream are not.

Reply:
But what about this?
```
$ion_1_1

$4 // name

(:values (:parse_ion '''$ion_1_1 $ion_literal::$ion_symbol_table::{symbols:["a", "b", "c", "d"]}''' ))

$4 // d
```
Are we okay with letting system values leak from `parse_ion` when it's wrapped in another macro/e-expression, and with the fact that `values` is no longer an identity function in this scenario?

Reply:
I've created https://github.com/amazon-ion/ion-docs/issues/365 to clarify this so as not to delay this PR.